ui/src/main/java/org/pentaho/di/ui/spoon/Spoon.java

```diff
@@ -4656,7 +4668,7 @@ public void openFileNew( String providerFilter, String fileFilterType, boolean i
         // We are able to find the index of forward or backward slash set the folder to be the path before slash
         String folder = lastFileOpened.substring( 0, parentIndex );
         fileDialogOperation.setPath( folder );
-        fileDialogOperation.setConnection( lastFileOpenedConnection );
```

Comment:
similar comment as https://github.com/pentaho/pentaho-kettle/pull/9310/files#r1594093304

variables are not used accessed by any code, so safe to set null.